OWNERS

```diff
@@ -1,11 +1,13 @@
+reviewers:
+ - cblecker
+ - mbarnes
+ - luis-falcon
+ - yithian
 approvers:
-- sre-platform-leads
 - cblecker
-- jewzaam
-- mwoodson
-- wanghaoran1988
-- dofinn
-- NautiluX
+- mbarnes
+- luis-falcon
+- srep-functional-leads
 reviewers:
```

Comment:
I can't get a suggestion to include the last row w/ mwoodson.

This reviewers and following lines should be removed.